packages/react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -7,82 +7,78 @@ import {
   getLastTabbable,
   getNextElement,
   focusAsync,
+  getPropsWithDefaults,
   modalize,
   on,
 } from '../../Utilities';
-import { useId, useConst, useMergedRefs, useUnmount } from '@fluentui/react-hooks';
+import { useId, useConst, useMergedRefs, useEventCallback, usePrevious, useUnmount } from '@fluentui/react-hooks';
 import { useDocument } from '../../WindowProvider';
 import type { IRefObject } from '../../Utilities';
 import type { IFocusTrapZoneProps, IFocusTrapZone } from './FocusTrapZone.types';
 
 interface IFocusTrapZoneInternalState {
-  disposeFocusHandler: (() => void) | undefined;
-  disposeClickHandler: (() => void) | undefined;
-  previouslyFocusedElementOutsideTrapZone: HTMLElement | undefined;
```

Comment:
I removed this from internalState because it only needed to be used within one effect and its cleanup (the equivalent variable is now called `elementToFocusOnDismiss`)